src/vs/workbench/contrib/webview/browser/pre/index.html

```diff
@@ -259,6 +259,8 @@
 						// service worker already loaded & ready to receive messages
 						postVersionMessage(currentController);
 					} else {
+						console.log(`Found unexpected service worker controller. Found: ${currentController?.scriptURL}. Expected: ${swPath}`);
```

Comment:
Should this console.log still be here?

Reply:
Yes, these were added to help us understand if a user is in this case which could be a sign of issues